doc/user_guide/installation.rst

```diff
@@ -1,27 +1,75 @@
 .. _installation:
 
 Installation
+============
+
+Command line
 ------------
 
+Pylint is installable using a package manager. Your package manager will find a version that
+works with your interpreter. We recommend ``pip``:
+
+.. code-block:: sh
+
+   pip install pylint
+
+
+Use the newer python interpreters if you can. It's possible to analyse code written
+for older interpreters by using ``py-version`` and setting it to the old interpreter. For
+example you can check that there are no ``f-string`` in python 3.5 code using python 3.8
+with an up-to-date pylint even if python 3.5 is past enf of life (EOL).
 
-Python packages
-'''''''''''''''
+If a newer version does not work for you, we do not maintain pylint for EOL python,
+so the best available pylint might be an old version that works with your old interpreter but
+without the bug fixes and feature of latest pylints.
 
-Pylint should be easily installable using pip.
+NB: You can also use ``conda`` or your system package manager on debian based OS.
+There package manager lag a little behind as it's maintained by a separate entity.
 
 .. code-block:: sh
 
-   pip install pylint
+   conda install pylint
+
+.. code-block:: sh
 
+   sudo apt-get install pylint
 
-Source distribution installation
-''''''''''''''''''''''''''''''''
 
-From the source distribution, extract the tarball, go to the extracted
-directory and simply run ::
+Editor integration
+------------------
 
-    python setup.py install
+.. _ide-integration:
 
-Or you can install it in editable mode, using ::
+- Eclipse_
+- Emacs_
+- `Eric IDE`_ in the `Project > Check` menu,
+- gedit_
+- `Jupyter Notebook`_
+- Komodo_
+- PyCharm_ :ref:`the section below <pylint_in_pycharm>`
+- PyDev_
+- pyscripter_ in the `Tool -> Tools` menu.
+- Spyder_ in the `View -> Panes -> Static code analysis`
+- TextMate_  :ref:`the section below <pylint_in_textmate>`
```

Comment:
```suggestion
- TextMate_, see our  :ref:`tutorial <pylint_in_textmate>`
```

Reply:
I removed the link to the IDE, if you're searching a pylint plugin for it you don't need to learn about it.